host-configs/blueos/clang_coral_2017_10_13-gpu.cmake

```diff
@@ -15,7 +15,6 @@
 set(RAJA_COMPILER "RAJA_COMPILER_CLANG" CACHE STRING "")
 
 set(CMAKE_CXX_COMPILER "/usr/tce/packages/clang/clang-coral-2017.10.13/bin/clang++-gpu" CACHE PATH "")
-set(CMAKE_C_COMPILER "/usr/tce/packages/clang/clang-coral-2017.10.13/bin/clang-gpu" CACHE PATH "")
 
 set(CMAKE_CXX_FLAGS_RELEASE "-O3 -fno-vectorize" CACHE STRING "")
```

Comment:
-fno-vectorize disables the vectorizer, do we want to have this in the release version?

Reply:
I think this may be to do with using OpenMP target?

Reply:
@artv3 @davidbeckingsale with this version of the compiler, the results are incorrect without this flag. I'm working on trying newer versions of the compiler to see if the issue goes away. But, I'm having link issues....  Also, one of the RAJA CUDA view tests is failing in the develop branch. Trying to track that down first.

Reply:
That make sense. Could you add a comment to the host-config file about the error.

Reply:
Sure. I'd like to remove that host-config and replace with more recent ones. But, unfortunately, that's the only one I can get thinks to link currently.

Reply:
@davidbeckingsale and @artv3 I built and ran the perf suite without '-fno-vectorize' flag and results were correct and there was no significant performance difference. I removed it and pushed the change. I may need re-approval.